wp-resources/plugins/sil-dictionary-webonary/templates/search-form.html

```diff
@@ -4,46 +4,58 @@
     }
 </style>
 <form name="searchform" id="searchform" method="get" action="@url@">
-    <input type="hidden" id="lang" name="lang" value="@qtrans_language@">
-    <input type="hidden" name="search_options_set" value="1">
-    <input id=displayAdvancedSearchId name="displayAdvancedSearchName" type="hidden" value="0">
+    <input type="hidden" id="lang" name="lang" value="@qtrans_language@" />
```

Comment:
Wow, I get to learn something new every day! Thanks :-)

But since XHTML standard dictates closing this tag, perhaps it's good for future to keep it as is? Also, FLex gives us XHTML, so to be consistent, perhaps we should use XHTML?
```
The <input> tag is empty, which means that the closing tag isn’t required. But in [XHTML](https://www.w3docs.com/learn-html/xhtml-extensible-hypertext-markup-language.html), the (<input>) tag must be closed (<input/>).
```

Reply:
Oh the closing tag was not something that I did intentionally, but vscode did, perhaps based on some default settings.